README.md

```diff
@@ -29,4 +29,20 @@ Currently I am exploring the following solutions:
 
 - using the CSS grid container itself to limit the display of rows (overflow will simply be hidden)
 
+## Solution
+
+This is one of the classical OMG moments, where the solution just hits you like a hammer.
+I was correct with removing the last child items through their index, but all I had to do is to keep the same index as the order of the children changed, in order to remove the correct pair of hands from the history. 
+
+Hence by calling the `.removeChild` function twice, but with the same index I make sure the sequence of child-removal is guaranteed. 
+
+```
+function cleanUpBottom(){
+    history.removeChild(history.children[12])
+    history.removeChild(history.children[12])
+}
+```
+
+This was a logical obstacle to overcome more than a programmatical. ✌😇
```

Comment:
It almost always is 😇

Bugs always have the stupidest solutions that make you say "fucccck that was all it was??"